src/astro/sql/__init__.py

```diff
@@ -59,6 +84,34 @@ def run_raw_sql(
     handler: Optional[Callable] = None,
     **kwargs: Any,
 ) -> TaskDecorator:
+    """
+    Given a python function that returns a SQL statement and (optional) tables, execute the SQL statement and output
+    the result into a SQL table.
+
+    Use this function as a decorator like so:
+
+      @transform
+      def my_sql_statement(table1: Table) -> Table:
+          return "DROP TABLE {{table1}}"
+
+    In this example, by identifying parameters as `Table` objects, astro knows to automatically convert those
+    objects into tables (if they are, for example, a dataframe). Any type besides table will lead astro to assume
+    you do not want the parameter converted.
+
+    Please note that the `run_raw_sql function will not create a temporary table. It will either return the result of a
```

Comment:
It is rst so need using double quotes not single for inline code formatting- ``run_raw_sql``